tests/test_feedexport.py

```diff
@@ -929,6 +929,114 @@ class MyItem2(scrapy.Item):
         yield self.assertExported(items, header, rows,
                                   settings=settings, ordered=True)
 
+    @defer.inlineCallbacks
+    def test_export_based_on_item_classes(self):
+        items = [
+            self.MyItem({'foo': 'bar1', 'egg': 'spam1'}),
+            self.MyItem2({'hello': 'world2', 'foo': 'bar2'}),
+            {'hello': 'world3', 'egg': 'spam3'},
```

Comment:
In case you plan to use these inputs for more tests, can we declare these as constants at the top?

Reply:
I'll keep this in mind next time I add tests.